pkg/extensions/sync/utils.go

```diff
@@ -37,7 +38,8 @@ type ReferenceList struct {
 // getTagFromRef returns a tagged reference from an image reference.
 func getTagFromRef(ref types.ImageReference, log log.Logger) reference.Tagged {
 	tagged, isTagged := ref.DockerReference().(reference.Tagged)
-	if !isTagged {
+	// hard to reach test case, injected error, see pkg/test/dev.go
+	if !test.Ok(isTagged) {
```

Comment:
This is only a warning log right? Why fault inject for this?

Reply:
yes, because I couldn't cover it otherwise :D, I can remove it if you want.